### DISCUSSION TITLE
Tutorial/Documentation uses/shows features of pydantic that throw an error

### DISCUSSION THREAD

**Discussion by RT-Tap** [fastapi/fastapi#4728] [Questions] [answered]
EDIT: DISREGARD:  I just found out I was using int's rather than strings and I can't delete this post now sorry.

Hey I didn't want to start a bug/issue thread so I figured I'd note it here.

It seems pydantic throws a value Error on constraints that are set but not enforced, for example if you want to add min_length or max_length to a Query as shown here: [https://fastapi.tiangolo.com/tutorial/query-params-str-validations/#additional-validation](url) you will get an error.  Just wanted to point it out to maybe remove from official documentation so people don't come around complaining.

Sample code you can try to verify:
```
 #!/usr/bin/python
from fastapi import FastAPI, Query
from typing import Optional # required for python <3.10
app = FastAPI()
@app.get("/")
async def root():
    return {"message": "Hello World"}
@app.get("/orderbook/{startTime}")
async def read_item(
        startTime: int = Query(..., min_length=9 ),#   max_length=11 
        endTime: Optional[int] = Query(None,min_length=9)):#   max_length=11
    if endTime:
        return {"startTime": startTime, 'endTime': endTime}
    return {"startTime": startTime}
```

**Comment by tiangolo** [comment, marked as answer]
Thanks for editing the original post and noting you found the solution. :nerd_face: 

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
